client/src/components/AdvisorReports/AdvisorReportsTable.js

```diff
@@ -1,13 +1,49 @@
-import { Settings } from "api"
+import API, { Settings } from "api"
+import { gql } from "apollo-boost"
 import AdvisorReportsRow from "components/AdvisorReports/AdvisorReportsRow"
 import AdvisorReportsTableHead from "components/AdvisorReports/AdvisorReportsTableHead"
+import { mapDispatchToProps, useBoilerplate } from "components/Page"
 import _uniqueId from "lodash/uniqueId"
 import PropTypes from "prop-types"
 import React from "react"
 import { Table } from "react-bootstrap"
+import { connect } from "react-redux"
+
+const GQL_GET_ADVISOR_REPORTS_INSIGHT = gql`
+  query($orgUuid: String!) {
+    advisorReportInsights(orgUuid: $orgUuid) {
+      uuid
+      name
+      stats {
+        week
+        nrReportsSubmitted
+        nrEngagementsAttended
+      }
+    }
+  }
+`
 
 const AdvisorReportsTable = props => {
```

Comment:
Maybe we can agree to extract the props in the beginning of each function (like you did with the BarChart), so that we also know quicker which props it has? Now I can go to the end and look at the propTypes to see that.

Reply:
In trying to minimise the changes (huh) I did not unfold the props everywhere. But as a convention we might adopt it. Could you make a debt issue to track this (and other JS style/conventions we want to follow)?